README.md

```diff
@@ -1,5 +1,5 @@
 <p align="center">
-    <a href="https://atomicdex.io" alt="Contributors">
+    <a href="https://komodoplatform.com/en/docs/komodo-defi-framework/api/v20/" alt="Contributors">
```

Comment:
When I first saw this I was like "when the hell did we get version 20?"; I think we should make it something like api/v2_0 to avoid confusion.

It has nothing to do with this PR btw.

Reply:
I'll open an issue in the docs repo - its  a pretty big structural change which likely would have some knock-on effects